extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -41,7 +39,8 @@ require([
 			plist: recommendedPlaylist,
 			vtags: videoTags
 		},
-		responseTimeout = 2000,
+		responseTimeout = 3000,
+		playerSetuped = false,
```

Comment:
It's going to be not needed if we reuse `waitForResponse` logic from ad-bidders package

Reply:
+1